docs/service_specs/census.md

```
@@ -10,36 +10,19 @@ The Census service is primarily responsible for maintaining a tenants admit and
-### Database Settings (MSSQL)
-
-| Name                      | Value                | Secret? |
-|---------------------------|----------------------|---------|
-| MongoDB__ConnectionString | `<ConnectionString>` | Yes     |
-| MongoDb__DatabaseName     | `<DatabaseName>`     | No      |
-| MongoDb__CollectionName   | `census`             | No      |
+See [Census Functionality](../functionality/census_management.md) for more information on the role of the Census service in the Link Cloud ecosystem.
+
+## Common Configurations
+
+* [Swagger](../config/csharp.md#swagger)
```

Markdownlint (0.37.0)

Expected: dash; Actual: asterisk
Unordered list style

(MD004, ul-style)